tests/unit/test_unit_asr.py

```diff
@@ -254,6 +254,47 @@ def test_kaldi_dataloader(self):
         )
         self.assertTrue(len(dl_test_max) == 19)
 
+    @pytest.mark.unit
+    def test_tarred_dataloader(self):
+        batch_size = 4
+        manifest_path = os.path.abspath(
+            os.path.join(os.path.dirname(__file__), '../data/asr/tarred_an4/tarred_audio_manifest.json')
+        )
+
+        # Test loading a single tarball
+        tarpath = os.path.abspath(os.path.join(os.path.dirname(__file__), '../data/asr/tarred_an4/audio_0.tar'))
```

Comment:
When is this data loaded? Is this what was added to the tests/data/asr.tar.gz file?

Reply:
Yes, I added a `tarred_an4` directory. It contains files `audio_{0..3}.tar` as well as the manifest used in the tests.